assets/xml/objects/object_famos.xml

```diff
@@ -1,7 +1,7 @@
 ﻿<Root>
     <!-- Death Armos: flying statues in Inverted Stone Tower Temple -->
     <File Name="object_famos" Segment="6">
-        <Animation Name="gFamosShakeAnim" Offset="0xF8" />
+        <Animation Name="gFamosShakeAnim" Offset="0xF8" /> <!-- Original name is "fam_aizu" -->
```

Comment:
Aizu like "Sign"? Makes sense the animation is there to tell you hes about to fall as a warning I guess.

Reply:
Yeah, I'm guessing it's something like "sign; signal; cue​". I should note that I've just been leaving these untranslated, because it significantly slows me down to look them up. There will be future passes with translations later, but others can help with that; for now, it seems I'm the only one who is looking at MM3D assets, so I might as well optimize the process to get through them quickly.